src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2ApprovedNotificationTest.java

```diff
@@ -260,5 +260,26 @@ void shouldSendEmailInWelshLanguageToApplicant1WhenRespondentApprovedAndLanguage
 
         verify(commonContent).getPartnerWelshContent(data, data.getApplicant2());
     }
+
+    @Test
+    void shouldSendEmailInWelshLanguageToApplicant1WhenHWFDeniedAndLanguagePreferenceIsWelsh() {
+        CaseData data = validApplicant2CaseData();
+        data.setDivorceOrDissolution(DIVORCE);
+        data.getApplication().getApplicant1HelpWithFees().setNeedHelp(YesOrNo.YES);
+        data.getApplication().getApplicant2HelpWithFees().setNeedHelp(YesOrNo.NO);
+        data.setDueDate(LOCAL_DATE);
+        data.getApplicant1().setLanguagePreferenceWelsh(YesOrNo.YES);
+
+        notification.sendToApplicant1(data, 1234567890123456L);
+
+        verify(notificationService).sendEmail(
+            eq(TEST_USER_EMAIL),
+            eq(JOINT_APPLICANT1_APPLICANT2_APPROVED_WITHOUT_HWF),
+            anyMap(),
```

Comment:
You could verify the map contents with:
            argThat(allOf(
                hasEntry(PARTNER, "gŵr")
            )),

and add a when before:

when(commonContent.getPartnerWelshContent(data, data.getApplicant2())).thenReturn("gŵr");